### DISCUSSION TITLE
load_model not support device_index

### DISCUSSION THREAD

**Discussion by zhanghongyu-git** [openai/whisper#1334] [Q&A] [answered]
hi, 
   i find that load_model function not support device_index, and WhisperModel support device_index,  but load_model function
not give device_index argument , because my computer has several GPU , so I want use device_index option ,
looking forward to the new version with this feature
thanks

**Comment by jongwook** [comment, marked as answer]
You can use the `device` argument to specify your device, such as `"cuda:3"`
